src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -0,0 +1,226 @@
+        ->alias(ParameterBagInterface::class, 'parameter_bag')
+
+        ->set('event_dispatcher', EventDispatcher::class)
+            ->public()
+            ->tag('container.hot_path')
+        ->alias(EventDispatcherInterfaceComponentAlias::class, 'event_dispatcher')
+        ->alias(EventDispatcherInterface::class, 'event_dispatcher')
+
+        ->set('http_kernel', HttpKernel::class)
+            ->public()
+            ->args([
+                service('event_dispatcher'),
+                service('controller_resolver'),
+                service('request_stack'),
+                service('argument_resolver'),
+            ])
+            ->tag('container.hot_path')
+        ->alias(HttpKernelInterface::class, 'http_kernel')
+
+        ->set('request_stack', RequestStack::class)
+            ->public()
+        ->alias(RequestStack::class, 'request_stack')
+
+        ->set('url_helper', UrlHelper::class)
+            ->args([
+                service('request_stack'),
+                service('router.request_context')->ignoreOnInvalid(),
+            ])
+        ->alias(UrlHelper::class, 'url_helper')
+
+        ->set('cache_warmer', CacheWarmerAggregate::class)
+            ->public()
+            ->args([
+                tagged_iterator('kernel.cache_warmer'),
+                param('kernel.debug'),
+                sprintf('%s/%sDeprecations.log', param('kernel.cache_dir'), param('kernel.container_class')),
+            ])
+            ->tag('container.no_preload')
+
+        ->set('cache_clearer', ChainCacheClearer::class)
+            ->public()
+            ->args([
+                tagged_iterator('kernel.cache_clearer'),
+            ])
+
+        ->set('kernel')
+            ->synthetic()
+            ->public()
+        ->alias(KernelInterface::class, 'kernel')
+
+        ->set('filesystem', Filesystem::class)
+            ->public()
+        ->alias(Filesystem::class, 'filesystem')
+
+        ->set('file_locator', FileLocator::class)
+            ->args([
+                service('kernel'),
+            ])
+        ->alias(FileLocator::class, 'file_locator')
+
+        ->set('uri_signer', UriSigner::class)
+            ->args([
+                param('kernel.secret'),
+            ])
+        ->alias(UriSigner::class, 'uri_signer')
+
+        ->set('config_cache_factory', ResourceCheckerConfigCacheFactory::class)
+            ->args([
+                tagged_iterator('config_cache.resource_checker'),
+            ])
+
+        ->set('dependency_injection.config.container_parameters_resource_checker', ContainerParametersResourceChecker::class)
+            ->args([
+                service('service_container'),
+            ])
+            ->tag('config_cache.resource_checker', ['priority' => -980])
+
+        ->set('config.resource.self_checking_resource_checker', SelfCheckingResourceChecker::class)
+            ->tag('config_cache.resource_checker', ['priority' => -980])
```

Comment:
`-990`

Reply:
Great catch. Fixed in ab739e5aa305cb1e8ef4404414006f392149d548